dallinger/experiment_server/experiment_server.py

```diff
@@ -421,19 +421,18 @@ def launch():
         )
 
     recruitment_details = None
-    if _config().get("open_recruitment"):
```

Comment:
Hi @pmcharrison 
Instead of removing this altogether, we should change it to:
`if _config().get("mode") == "live":`
This will prevent creating studies/surveys/HITs and also solve the issue we have with starting Prolific in local debug mode. Makes sense?

Reply:
Does that work even for MTurk sandbox, though? There we do open recruitment, but it's just in the MTurk sandbox.

Reply:
I see, then we just add `sandbox`, like:
`if _config().get("mode") in {"live", "sandbox"]:`
?

Reply:
Made this change now:
https://github.com/Dallinger/Dallinger/pull/7363/commits/ce8e47c95ba0d8a57e118ce3165fafb5ed5b07d2

Reply:
@fmhoeger our cardinal rule should be that we do not hardcode recruiter-specific logic outside the recruiter class.

My suggestion instead would be to always cal open_recruitment, but then for ProlificRecruiter.publish_study return early if mode is not live.

Reply:
OK, I reverted these changes regarding `open_recruitment`. The PR should be ready: Please check again.